src/components/pages/Categories/index.vue

```diff
@@ -67,6 +84,21 @@ export default {
     updateValue(event) {
       this.targetCategory.name = event.target.value;
     },
+    handleClick() {
+      const categoryId = this.$data.deleteCategory.id;
```

Comment:
[must]
$dataを経由しなくても参照できるので修正をお願いします！
```suggestion
      const categoryId = this.deleteCategory.id;
```

他にも$dataを使っているところは全て書き換え可能なので、合わせて修正をお願いします！

Reply:
「$data」を使っている箇所について、修正いたしました。[eee2da0](https://github.com/gizumo-education/gizumo_wiki_lesson/pull/112/commits/eee2da0cf144c744fd80909ed56dab3facfc5f8e)